src/sidebar/util/test/group-organizations-test.js

```diff
@@ -0,0 +1,174 @@
+var orgFixtures = require('../../test/group-fixtures');
+
+describe('group-organizations', function () {
+  context ('when sorting organizations and their contained groups', function () {
+
+    it ('should put the default organization groups last', function () {
+      const defaultOrg = orgFixtures.defaultOrganization();
+      const groups = [orgFixtures.expandedGroup({ organization: defaultOrg }),
+                      orgFixtures.expandedGroup(),
+                      orgFixtures.expandedGroup()];
+
+      const sortedGroups = groupsByOrganization(groups);
+
+      assert.equal(sortedGroups[2].organization.id, defaultOrg.id);
+    });
+
+    it ('should sort organizations by name', function () {
+      const org1 = orgFixtures.organization({ name: 'zzzzz' });
+      const org2 = orgFixtures.organization({ name: 'aaaaa' });
+      const org3 = orgFixtures.organization({ name: 'yyyyy' });
+      const groups = [orgFixtures.expandedGroup({ organization: org1 }),
+                      orgFixtures.expandedGroup({ organization: org2 }),
+                      orgFixtures.expandedGroup({ organization: org3 })];
+
+      const sortedGroups = groupsByOrganization(groups);
+
+      assert.equal(sortedGroups[0].organization.name, 'aaaaa');
+      assert.equal(sortedGroups[1].organization.name, 'yyyyy');
+      assert.equal(sortedGroups[2].organization.name, 'zzzzz');
+    });
+
+    it ('should sort organizations secondarily by id', function () {
+      const org1 = orgFixtures.organization({ name: 'zzzzz', id: 'zzzzz' });
+      const org2 = orgFixtures.organization({ name: 'zzzzz', id: 'aaaaa' });
+      const org3 = orgFixtures.organization({ name: 'zzzzz', id: 'yyyyy' });
+      const groups = [orgFixtures.expandedGroup({ organization: org1 }),
+                      orgFixtures.expandedGroup({ organization: org2 }),
+                      orgFixtures.expandedGroup({ organization: org3 })];
+
+      const sortedGroups = groupsByOrganization(groups);
+
+      assert.equal(sortedGroups[0].organization.id, 'aaaaa');
+      assert.equal(sortedGroups[1].organization.id, 'yyyyy');
+      assert.equal(sortedGroups[2].organization.id, 'zzzzz');
+    });
+
+    it ('should only include logo for first group in each organization', function () {
+      const org = orgFixtures.organization({ name: 'Aluminum' });
+      const org2 = orgFixtures.organization({ name: 'Zirconium' });
+      const groups = [
+        { name: 'Aluminum', organization: org },
+        { name: 'Beryllium', organization: org2},
+        { name: 'Butane', organization: org2 },
+        { name: 'Cadmium', organization: org},
+      ];
+
+      const sortedGroups = groupsByOrganization(groups);
+
+      assert(typeof sortedGroups[0].logo === 'string');
+      assert(typeof sortedGroups[1].logo === 'undefined');
+      assert(typeof sortedGroups[2].logo === 'string');
+      assert(typeof sortedGroups[3].logo === 'undefined');
+
```

Comment:
Extra blank line

Reply:
Is there a way we could add this to linting checks if it's a consistency we want to enforce?

Reply:
I'm afraid I don't know if ESLint has a suitable rule offhand and I couldn't immediately find it. If we can, I'd be happy to have it enabled.